### PR TITLE
Small spec-text fix

### PR DESCRIPTION
**Pull request by jridgewell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ When the SetLastArrayItem method is called, the following steps are taken:
 
     Let O be ? ToObject(this value).
     Let len be ? ToLength(? Get(O, "length")).
-    If len is zeo, then
+    If len is not zero, then
```
Comment (ljharb) [inline README.md]: maybe `> 0`?

Comment (jridgewell) [reply README.md]: 🤷‍♂️, how does the rest of the spec handle it? Isn't "zero or positive" implied by `ToLength`?

Comment (ljharb) [reply README.md]: good point. I think it is, but I think that making implications explicit might still be helpful.

 - https://tc39.github.io/ecma262/#sec-string.raw uses `<= 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.includes uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.indexof uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.lastindexof uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.pop uses `is zero`
 - https://tc39.github.io/ecma262/#sec-array.prototype.reduce uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.reduceright uses `is 0`
 - https://tc39.github.io/ecma262/#sec-array.prototype.shift uses `is zero`

All the other use cases I found were part of a more generic algorithm, like a loop.

Sounds like `is 0` is the overarching convention. I'll make a PR to convert the remainder in the spec.

Comment (ljharb) [reply README.md]: https://github.com/tc39/ecma262/pull/1086

Comment (jridgewell) [reply README.md]: Updated.

**Review by ljharb — APPROVED** [review]

**Comment by keithamus** [comment]
Thanks both 👍 ❤️

**Pull request merged**